security/cmd/istio_ca/main.go

```diff
@@ -274,6 +280,23 @@ func runCA() {
 				Namespace:   opts.istioCaStorageNamespace,
```

Comment:
And since you are already here, consolidate this webhookServiceNames/webhookServiceAccounts with the customerDNSNames?

Reply:
Separate PR maybe ? Didn't want to add risks to 0.8 by touching code that is used. We can cleanup in 0.9.